scripts/ci_prepare_python.bash

```diff
@@ -10,30 +10,6 @@ sudo add-apt-repository -y ppa:deadsnakes/ppa
 sudo apt-get install python3.5 python3.5-dev
 sudo apt-get install python3.7 python3.7-dev
 sudo apt-get install python3.8 python3.8-dev
+sudo apt-get install python3.9 python3.9-dev
 sudo apt-get install tox
 pip3 install wheel setuptools
-
-pushd haskell
-
-docker build --tag haskell-sbp2json .
-
-docker run \
-    -v "$PWD:/work" \
-    -v "$HOME/.docker-cache/stack/work:/work/.stack-work" \
-    -v "$HOME/.docker-cache/stack/home:/root/.stack" \
-    --rm \
-    --name haskell-sbp2json-run \
-    haskell-sbp2json \
-    stack build -j4 --allow-different-user
-
-sudo chown "$USER:$USER" "$HOME/.docker-cache"
-sudo chown -R "$USER:$USER" "$HOME/.docker-cache/stack"
-sudo chown -R "$USER:$USER" "$PWD"
-
-haskell_bins="$(find "$HOME/.docker-cache/stack/work/install" -name bin)"
-
-cp "$haskell_bins/sbp2json" .
-cp "$haskell_bins/sbp2prettyjson" .
-cp "$haskell_bins/json2sbp" .
-cp "$haskell_bins/json2json" .
-cp "$haskell_bins/sbp2nmea" .
```

Comment:
Aren't these binaries used for the benchmarks?

Reply:
Hmm... `ci_benchmark.bash` does refer to three of them. We'll see if it fails.  We can have a separate script with these for benchmark only, as they are not needed for the Python test job.

Reply:
Looks like the benchmarks passed and from looking at the CI output it looks like the benchmark phase built the binaries.  I'll wait for @notoriaga to chime in since he mostly recently looked through that code.

Reply:
Yeah I think it build's them separately. I think initially I tried to share artifacts across jobs but it didn't end up working. I think this should be alright